cura/CrashHandler.py

```diff
@@ -83,6 +85,20 @@ def __init__(self, exception_type, value, tb, has_started = True):
         self.dialog = QDialog()
         self._createDialog()
 
+    @staticmethod
+    def pruneSensitiveData(obj):
+        if type(obj) is list:
```

Comment:
I think it's a bit better to use isinstance instead (eg; It might happen that you get an ordered dict)